app/Models/User.php

```diff
@@ -300,13 +300,13 @@ public static function validateUsername($username, $previousUsername = null)
 
             if ($remaining->days > 365 * 2) {
                 //no need to mention the inactivity period of the account is actively in use.
-                return ['Username is already in use!'];
+                return [trans('model_validation.user.username_in_use')];
             } elseif ($remaining->days > 0) {
-                return ["This username will be available for use in <strong>{$remaining->days}</strong> days."];
+                return [trans_choice('model_validation.user.username_available_in_days', $remaining->days)];
             } elseif ($remaining->h > 0) {
-                return ["This username will be available for use in <strong>{$remaining->h}</strong> hours."];
+                return [trans_choice('model_validation.user.username_available_in_hours', $remaining->hours)];
```

Comment:
The property name used between here and the check doesn't match ಠ_ಠ

Or more like `->hours` doesn't exist.